zilliqa/src/message.rs

```diff
@@ -196,6 +212,18 @@ pub struct BlockRequest {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct BlockResponse {
     pub proposals: Vec<Proposal>,
+    pub from_view: u64,
+    /// When we send a block response, we may also send data on what blocks we are prepared
+    /// to serve.
+    pub availability: Option<Vec<BlockStrategy>>,
+}
+
+/// Used to convey proposal processing internally, to avoid blocking threads for too long.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ProcessProposal {
+    // An encoded PeerId
+    pub from: Vec<u8>,
```

Comment:
Because I wanted to force the representation of fear of incompatibility if PeerId (which is in an external library) changed its external representation; very happy to change it?